editor/src/clj/util/coll.clj

```diff
@@ -767,3 +898,48 @@
    (transduce identity str-rf coll))
   ([sep coll]
    (transduce (interpose (str sep)) str-rf coll)))
+
+(defn consensus
```

Comment:
That's an unexpected name... I don't think the operation is generic enough to be in `coll`, maybe leave it in `shader-gen` where it's used?

Reply:
Is it? There are multiple definitions, but these seem quite close to what I was going for:
* An agreement between two or more parties on an issue
* General voice speaking as one accord

I think it has utility as a general function in `coll`, but the name could maybe be better.
* `unanimity` is "A state of being in unanimous agreement by all involved parties".
* `unity` is, amongst other things, "The state or condition of being equal, identical or alike".
* `unison` is "The state of acting together, at the same time, or as one", "The quality of forming a pleasing and consistent whole", "The state or quality of being identical", and "General voice speaking as one accord". This one seems pretty good to me. 🤔

Reply:
Everything is equal... Socialism 🤪

Reply:
`coll/socialism` is tempting, but I think I'll go with `unanimous-value` for now! 😅